include/asterisk/res_pjsip.h

```diff
@@ -87,6 +87,8 @@
 #define AST_STIR_SHAKEN_RESPONSE_STR_UNSUPPORTED_CREDENTIAL "Unsupported Credential"
 #define AST_STIR_SHAKEN_RESPONSE_STR_INVALID_IDENTITY_HEADER "Invalid Identity Header"
 
+#define PJSIP_AUTH_MAX_REALM_LENGTH 255	/* From the auth/realm realtime column size */
```

Comment:
We should try to avoid prefixing things with `PJSIP_` in case the upstream project wanted to use it, `AST_SIP_AUTH_MAX_REALM_LENGTH` instead maybe?

Reply:
Sure, and I believe it's crucial to ensure the commit message accurately reflects the intent of this change. I've updated the message accordingly.